1-js/08-prototypes/02-function-prototype/article.md

```diff
@@ -150,26 +150,27 @@ Rabbit.prototype = {
 */!*
 };
 
-// now constructor is also correct, because we added it
+// agora o constructor também está correto, porque nós o adicionamos
 ```
 
 
-## Summary
+## Resumo
 
-In this chapter we briefly described the way of setting a `[[Prototype]]` for objects created via a constructor function. Later we'll see more advanced programming patterns that rely on it.
+Neste capítulo, nós descrevemos brevemente a forma de configurar um `[[Prototype]]` para os objetos criados via função construtura. Mais tarde nós vamos ver padrões (*patterns*) mais avançados de programação que dependem disso.
 
-Everything is quite simple, just a few notes to make things clear:
+É tudo bem simples, mas aqui estão algumas notas para deixar as coisas claras:
 
-- The `F.prototype` property (don't mistake it for `[[Prototype]]`) sets `[[Prototype]]` of new objects when `new F()` is called.
-- The value of `F.prototype` should be either an object or `null`: other values won't work.
--  The `"prototype"` property only has such a special effect when set on a constructor function, and invoked with `new`.
+- A propriedade `F.prototype` (não confunda com o `[[Prototype]]`) configura o `[[Prototype]]` de novos objetos quando `new F()` é chamado.
+- O valor de `F.prototype` deveria ser um objeto ou `null`: outros valores não vão funcionar.
+- A propriedade `"prototype"` só tem o efeito especial quando configurada em uma função construtora, e invocada com `new`.
+
+Em objetos regulares, o `prototype` não tem nada de especial:
 
-On regular objects the `prototype` is nothing special:
 ```js
 let user = {
   name: "John",
-  prototype: "Bla-bla" // no magic at all
+  prototype: "Bla-bla" // nenhuma mágica aqui
 };
 ```
 
-By default all functions have `F.prototype = { constructor: F }`, so we can get the constructor of an object by accessing its `"constructor"` property.
+Por definição, todas as funções possuem `F.prototype = { constructor: F }`, então nós podemos obter o construtor de um objeto acessando sua propriedade `"constructor"`.
```

Comment:
> Expressões prolixas - [PT_WORDINESS_REPLACE](https://community.languagetool.org/rule/show/PT_WORDINESS_REPLACE?lang=pt)

- `Por definição` is a long-winded expression.
- Suggestion: change to `Por padrão`

<img width="671" alt="image" src="https://user-images.githubusercontent.com/35238611/177465244-afdc1a98-1673-40e0-8ea6-db202aebe226.png">